testsuite/tests/tool-ocamldoc/Inline_records.man.reference

```diff
@@ -1,3 +1,4 @@
+.TH "Inline_records" 3 2019-09-02 OCamldoc ""
```

Comment:
The CI failure is probably due to this line, `ocamltest -promote` is not aware that the ocamldoc's output comparison skip the first line of the output.